### ISSUE TITLE
authority of species and synonym

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#1203]
[Originally posted on GoogleCode (id 1025) on 2012-01-28]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/6268

<b>What data are incorrect or missing?</b>
Juncus castaneus subsp. leucochlamys (N.W. Zinger ex V.I. Kreczetowicz) Hultén
Juncus leucochlamys Zinger ex V.I. Kreczetowicz

<b>What data are you expecting instead?</b>
Juncus castaneus subsp. leucochlamys (V[W].J. Zinger ex V.I. Kreczetowicz) Hultén
Juncus leucochlamys V[W].J.Zinger ex V.I. Kreczetowicz

<b>If applicable, please provide an authoritative source.</b>
I'm not sure which is correct, but the PanArctic Flora uses V.J Zinger..., as does The Plant List.  
see: http://www.nhm.uio.no/english/research/infrastructure/paf/320106 

But if V.J. Zinger is correct, IPNI authors uses W rather than V; Here are their only 2 entries for Zinger:

```
N.W.Zinger - Nikolai Wasiljevicz Zinger 1866-1923

W.J.Zinger - Wasili Jakowlewitsch Zinger 1836-1907
```

http://www.nhm.uio.no/english/research/infrastructure/paf/320106

**Comment by brouille** [comment]
SEnt to K. Ghandi May 3, 2020:
Kanchi :

This was originally brought to my attention by Sue Meades in 2012, but I hesitated to bring it to your attention because of the FNA overload at the time and since.

This concerns the names J. castaneus subsp. leucochlamys (N. W. Zinger. ex V. I. Kreczetowicz) Hultén; J. leucochlamys N. W. Zinger ex V. I. Kreczetowicz
 Which have been placed in synonymy of Juncus castaneus in FNA:
J. castaneus subsp. leucochlamys (N. W. Zinger. ex V. I. Kreczetowicz) Hultén; J. leucochlamys N. W. Zinger ex V. I. Kreczetowicz
http://www.efloras.org/florataxon.aspx?flora_id=1&taxon_id=222000110
The authors are the same as in TROPICOS: legacy.tropicos.org/name/16900529

This taxon is recognized as valid by Kirschner 2002 (see https://wcsp.science.kew.org/namedetail.do?name_id=314737) and would be present in northwestern NA (AK-YU-NT). Here, however, the authorities are listed as Juncus castaneus subsp. leucochlamys (V.J.Zinger ex V.I.Krecz.) Hultén
Many other sources use the same initials (obviously, most associated to Kew such as https://wcvp.science.kew.org/taxon/77188342-1 and POWO), as well as the PAF: http://panarcticflora.org/results?biogeographic=&bioclimatic=&region=&name=Juncus+castaneus#paf-320106

And also including IPNI: https://www.ipni.org/n/77188342-1
However, IPNI https://www.ipni.org/n/443131-1 gives Zing. ex V.I. Kreck. for the basionym J. leucochlamys. Without initials.

Zing. without initials is not listed among authors in IPNI
N.W.Zinger – Nikolai Wasiljevicz Zinger (1866-1923) 
W.J.Zinger – Wasili Jakowlewitsch Zinger (1836-1907) [also V.J. – which is correct?]


The difficulty here lies with the initials of Zinger and the time of publication.

TROPICOS does not provide a citation for the basionym J. leucochlamys. IPNI provides de following citation:
Juncus leucochlamys Zing. ex V.I.Krecz., Fl. Transbaical. ii. 141 (1931); Krecz. & Gontsch. in Komarov, Fl. URSS, iii.526, 627 (1935).
https://www.ipni.org/n/443131-1

I have not been able to access Fl. Transbaical., which is not listed in IPNI under publications either.
Kreczetowicz appears to have attributed the epithet leucochlamys to one or the other Zinger. The work being published in 1931 would be compatible with both N.W. (d 1923) and V.J. (d. 1907), and thus publication date appears a tenuous way to solve the issue, though the former is closer in date to Krecz.

Since this affects FNA, I am submitting the issue to you, in the hope that you will be able to eventually determine which is correct.

Best
Luc

**Comment by brouille** [comment]
Response frrom K. Gandhi, email 3-05-2020
"IPNI’s standard abbreviation:
Wasili Jakowlewitsch Zinger (1836-1907): W.J.Zinger [Alternative Names: Vasili Yakovlevich Zinger & Vasilij Jakovlevič, Zinger]
I just revised the IPNI authorship to show:
1) Juncus leucochlamys W.J.Zinger ex V.I.Krecz. Fl. Transbaical. ii. 141 (1931); Krecz. & Gontsch. in Komarov, Fl. URSS, iii.526, 627 (1935).
2) Juncus castaneus subsp. leucochlamys (W.J.Zinger ex V.I.Krecz.) Hultén, Ark. Bot. ser. 2, 7(1): 32. 1968
I do NOT know whether 1931 publ. is valid or not."

Authorities corrected in VASCAN

**Comment by sjmeades** [comment]
Thanks, Luc.

 

Hope you are doing well.  I see you’re keeping busy with FNA business.  I look forward to seeing that next volume when it’s published.  

 

I got a message from Howard Clase a short while ago and he asked if I had heard back from you re: the Pilosella specimens I sent you last year.  He believes some are P. flagellaris, as he’s familiar with that species from England. 

I also wondered if you were able to get an ID on the specimen I sent you from Batchawana Island, Algoma District, nON.  It was a sterile specimen with alternate, oblanceolate leaves and obvious rounded buds in the leaf axils.  There were very many specimens of that on the Island growing on the beaches.  I wondered if it might be a sterile specimen of Comandra umbellata, judging from the leaf shape and size of the plant.  It was  collected in mid-Sept. of 2018.  Of the hundreds of plants I saw, I didn’t see any signs of fruits.

 

Stay safe and take care, Sue.

**Comment by brouille** [comment]
Thanks Sue

Concerning the Clase specimen : I have trouble determining it due to the quality of the specimen. Some characters missing. And obviously, I lost it in the mess of things to do to finish FNA volumes and my Asteraceae for Flore nordique du Québec-Labrador. All done now.

I am not sure Geoff has given me the specimens from Algoma. And I cannot check now, of course. This will have to await the reopening of the university (and permission to get in since I am over 65).

Luc

De : sjmeades <notifications@github.com>
Répondre à : Canadensys/vascan-data <reply@reply.github.com>
Date : lundi 4 mai 2020 à 10:37
À : Canadensys/vascan-data <vascan-data@noreply.github.com>
Cc : Luc Brouillet <luc.brouillet@umontreal.ca>, State change <state_change@noreply.github.com>
Objet : Re: [Canadensys/vascan-data] authority of species and synonym (#1203)

Thanks, Luc.



Hope you are doing well. I see you’re keeping busy with FNA business. I look forward to seeing that next volume when it’s published.



I got a message from Howard Clase a short while ago and he asked if I had heard back from you re: the Pilosella specimens I sent you last year. He believes some are P. flagellaris, as he’s familiar with that species from England.

I also wondered if you were able to get an ID on the specimen I sent you from Batchawana Island, Algoma District, nON. It was a sterile specimen with alternate, oblanceolate leaves and obvious rounded buds in the leaf axils. There were very many specimens of that on the Island growing on the beaches. I wondered if it might be a sterile specimen of Comandra umbellata, judging from the leaf shape and size of the plant. It was collected in mid-Sept. of 2018. Of the hundreds of plants I saw, I didn’t see any signs of fruits.



Stay safe and take care, Sue.
